Script Analysis Examples/Specification Analysis/Simple Specification Analysis Example.ipynb

```diff
@@ -0,0 +1,245 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple Specification Analysis Example\n",
+    "\n",
+    "This example demonstrates a simple analysis for various specs inside products. It uses the **Spec Service** to query, analyze and update the specs with the latest properties."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Imports\n",
+    "\n",
+    "Import Python Modules for executing the notebook. The **nisystemlink** library is used for communicating with various SystemLink Enterprise's endpoints. Scrapbook is used for running notebooks and recording data for the SystemLink Notebook Execution Service.\n",
+    "\n",
+    "The SYSTEMLINK_API_KEY environment variable specifies an API key created for the user executing this notebook, which provides Role Based Access Control to the various SystemLink APIs called by this notebook. The API key will expire after 24 hours.\n",
```

Comment:
See my comment below about using the jupyter configuration for the clients. We can then remove the markdown here about the api key.

Reply:
Removed the markdown